### ISSUE TITLE
Submit button disabled after file upload type not allowed

### ISSUE THREAD

**Issue by harrykc** [CalderaWP/Caldera-Forms#722]
Thanks for the great plugin.

I created a simple form with 3 elements.
1. File -> Allow Multiple checked, Allowed Types is "png"
2. Advanced File Uploader -> Allow Multiple checked, Allowed Types is "png"
3. submit button

There are 3 issues.
1. When click "Add Files", the select image window will show 2 times
2. When add png file into File field, then add another png file into "Advanced File Uploader" field, all the png files in File field will add into "Advanced File Uploader" field
3. When add pdf file into File field, then submit, it will say file type incorrect, then i change it to png file or keep pdf file, submit again, the submit button will disabled and can not be function anymore.

**Comment by Shelob9** [comment]
@harrykc  Which version of Caldera Forms are you using? We have made a lot of improvements to advanced file fields in version 1.4, which is currently in beta but should be released tomorrow.

**Comment by harrykc** [comment]
@Shelob9 the version I am using is 1.3.5.3. So i will update the status of this issue after 1.4 released. Thanks.

**Comment by harrykc** [comment]
Thanks for update.
Updated to Version 1.4.1 and **following issues will happened when multiple uploader in same form**.

I created a simple form with 3 elements.
1. File -> Allow Multiple checked, Allowed Types is "png"
2. Advanced File Uploader -> Allow Multiple checked, Allowed Types is "png"
3. submit button

Issues:
1. When click "Add Files" on File field, the select image window will show 2 times
2. When add file into File field, selected file will not insert into the view.
3. When add pdf file into Advanced File field, then submit, it will say file type incorrect, then i change it to png file or keep pdf file, submit again, the submit button will disabled and can not be function anymore.
